lib/db/mysql/patches/patch-005-006.sql

```diff
@@ -14,7 +14,7 @@ CREATE TABLE refreshTokens (
 
 -- Add expiresAt column for access tokens
 
-ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP NOT NULL;
+ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
Unrelatedly, this patch was failing to apply on my local mysql, complaining about creating a non-null column with no default value.